applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -0,0 +1,97 @@
+.. _nrf_desktop_dfu_mcumgr:
+
+Device Firmware Upgrade MCUmgr module
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Device Firmware Upgrade MCUmgr module is responsible for performing a Device Firmware Upgrade (DFU) over Simple Management Protocol (SMP).
+
+You can perform the DFU using, for example, the `nRF Connect for Mobile`_ application.
+The :guilabel:`DFU` button appears in the tab with the connected Bluetooth® devices.
+After pressing the button, you can select the :file:`*.bin` file used for the firmware update.
```

Comment:
Please remember to add link to general nRF Desktop doc here that informs which file to use (may be done as follow-up PR).

Reply:
moved to follow-up